test/SlowTests/Bugs/Caching/UseCachingInLazyTests.cs

```diff
@@ -422,6 +421,66 @@ await using (var bulk = store.BulkInsert())
             }
         }
 
+        [Fact]
+        public async Task LazilyLoad_WhenUseNotFoundCacheItem_ShouldUseItReleaseAtTheEnd()
+        {
+            const string notExistDocId = "NotExistDocId";
+
+            using (var store = GetDocumentStore())
+            {
+                store.AggressivelyCache();
+
+                using (var session = store.OpenAsyncSession())
+                {
+                    //Add "NotExistDocId" to cache
+                    await session.Advanced.Lazily.LoadAsync<TestObj>(notExistDocId).Value;
+                }
+
+                using (var session = store.OpenAsyncSession())
+                {
+                    //Add "NotExistDocId" to cache
+                    await session.Advanced.Lazily.LoadAsync<TestObj>(notExistDocId).Value;
+                    Assert.Equal(0, session.Advanced.NumberOfRequests);
+                }
+            }
+
+#if DEBUG
+            await Task.Delay(10000);
```

Comment:
I found it the best way to test it without modifying the production code for the test and without relying too much on the internal implementation.

Reply:
Let's remove this. The whole test suite will die anyway.